tensorboard/uploader/exporter_test.py

```diff
@@ -639,16 +639,48 @@ def test_experiments_only(self):
         def stream_experiments(request, **kwargs):
             del request  # unused
             response = export_service_pb2.StreamExperimentsResponse()
-            response.experiments.add(experiment_id="789", name="one")
-            response.experiments.add(experiment_id="012", description="two")
+            response.experiments.add(
+                experiment_id="789",
+                name="one",
+                num_scalars=8,
+                total_tensor_bytes=1234,
+                total_blob_bytes=0,
+                num_runs=1,
+                num_tags=10,
+            )
```

Comment:
What are these test changes for? It looks like all the prod code changes
are in `uploader_main.py`, which isn’t covered by `exporter_test`, so I
don’t see why they’re relevant. (All tests pass either with or without
these changes.) I also don’t see why the changes are desirable: the RPC
path in this test invokes `StreamExperiments` with a fieldmask that does
not include these extra metadata fields, so why change the server to
populate them?

Reply:
Reverted the changes to this file.